src/kerko/extractors.py

```diff
@@ -375,6 +377,12 @@ def extract(self, item, library_context, spec):
             if tag and \
                     (not self.include or self.include.match(tag)) and \
                     (not self.exclude or not self.exclude.match(tag)):
+                if self.include:
+                    match = self.include.match(tag)
+                    if match:
+                        # Check if there are capturing groups
+                        if len(match.groups()) > 0:
+                            tag = ''.join(match.groups())
```

Comment:
I'm afraid this could be problematic if the regular expression happens to have multiple or nested groups. It looks like the user would need a way to tell the extractor which group to use (group index, or group name, for regular expressions in the form of `(?P<name>...)`).